front/app/containers/EventsShowPage/components/EventShowPageMeta.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+
+import { Helmet } from 'react-helmet-async';
+
+import { IEventData } from 'api/events/types';
+import useEventImage from 'api/event_images/useEventImage';
+
+import useAppConfigurationLocales from 'hooks/useAppConfigurationLocales';
+import useLocalize from 'hooks/useLocalize';
+
+import messages from '../messages';
+
+import { useIntl } from 'utils/cl-intl';
+import getAlternateLinks from 'utils/cl-router/getAlternateLinks';
+import getCanonicalLink from 'utils/cl-router/getCanonicalLink';
+import { imageSizes } from 'utils/fileUtils';
+import { stripHtml } from 'utils/textUtils';
+
+interface Props {
+  event: IEventData;
+}
+
+const EventShowPageMeta = ({ event }: Props) => {
+  const { formatMessage } = useIntl();
+  const localize = useLocalize();
+  const tenantLocales = useAppConfigurationLocales();
+  const { data: eventImage } = useEventImage(event);
+
+  if (!tenantLocales) return null;
+
+  const metaTitle = formatMessage(messages.metaTitle, {
+    eventTitle: localize(event.attributes.title_multiloc, {
+      maxChar: 50,
+    }),
+  });
+  const description = stripHtml(
+    localize(event.attributes.description_multiloc),
+    250
+  );
+  const image = eventImage?.data?.attributes?.versions?.large;
+  const { location } = window;
+
+  return (
+    <Helmet>
+      <title>{metaTitle}</title>
```

Comment:
`ProjectShowPageMeta` has this code for `<title>`:
```
<title>
    {`${
      authUser &&
      typeof authUser.data.attributes.unread_notifications === 'number' &&
      authUser.data.attributes.unread_notifications > 0
        ? `(${authUser.data.attributes.unread_notifications}) `
        : ''
    }
        ${metaTitle}`}
  </title>
 ```
 I can of guessed this didn't make sense for events, but I'm not exactly sure it doesn't 🤔